index.js

```diff
@@ -54,18 +63,23 @@ client.on('ready', () => {
 client.on('messageCreate', initialQuery => {
     if (initialQuery.author.bot) return;
   
-    if (initialQuery.content.indexOf('!s ') == 0) {
+    if ( config.AllowConfigDump === true &&  initialQuery.content.indexOf('!configDump') === 0) {
```

Comment:
Set `ALLOW_CONFIG_DUMP to true` and then you got this command. Don't do this in production.